adafruit_button.py

```diff
@@ -59,12 +65,68 @@ class Button(displayio.Group):
     :param selected_outline: Inverts the outline color.
     :param selected_label: Inverts the label color.
     """
+    RECT: int = const(0)
+    ROUNDRECT: int = const(1)
+    SHADOWRECT: int = const(2)
+    SHADOWROUNDRECT: int = const(3)
+
+    def __init__(
+        self,
+        *,
+        x: int,
+        y: int,
+        width: int,
+        height: int,
+        name: Optional[str] = None,
+        style: int = RECT,
+        fill_color: Optional[int] = 0xFFFFFF,
+        outline_color: Optional[int] = 0x0,
```

Comment:
I think fill and outline colors can use tuple colors in addition to hex. I'm not positive though, I'll try to doublecheck.

Reply:
That was correct, both `fill_color` and `outline_color` can be tuples in addition to ints.

Reply:
I missed that! Added.